FlowCryptUI/Nodes/PassPhraseAlertNode.swift

```diff
@@ -182,26 +188,30 @@ public class PassPhraseAlertNode: ASDisplayNode {
         let font = isBold ? UIFont.boldSystemFont(ofSize: fontSize) : UIFont.systemFont(ofSize: fontSize)
         node.attributedText = NSAttributedString(
             string: text,
-            attributes: [NSAttributedString.Key.font: font]
+            attributes: [
+                .font: font,
+                .foregroundColor: Constants.textColor
+            ]
         )
+
         node.accessibilityIdentifier = identifier
         return node
     }
 
     private func createButtonNode(title: String, color: UIColor, identifier: String, action: Selector) -> ASButtonNode {
         let node = ASButtonNode()
-        node.setTitle(title, with: UIFont.systemFont(ofSize: 15), with: color, for: .normal)
+        node.setTitle(title, with: Constants.buttonFont, with: color, for: .normal)
         node.style.flexGrow = 1
         node.style.preferredSize.height = 35
         node.addTarget(self, action: action, forControlEvents: .touchUpInside)
         node.accessibilityIdentifier = identifier
-        node.setBackgroundColor(.lightGray, forState: .highlighted)
+        node.setBackgroundColor(UIColor.colorFor(darkStyle: .darkGray, lightStyle: .lightGray), forState: .highlighted)
         return node
     }
 
     private func updateIntroduction() {
         if let introduction, !introduction.isEmpty {
-            introductionLabel.attributedText = NSAttributedString(string: introduction)
+            introductionLabel.attributedText = NSAttributedString(string: introduction, attributes: [.foregroundColor: Constants.textColor])
```

Comment:
there is an issue with color change when iPhone switches between light and dark modes:
1) open message which needs passphrase
2) passphrase alert is shown, colors are ok
3) click `Cancel` and go back to inbox screen
3) go to simulator settings, switch from light mode to dark (or vice versa)
4) go to FlowCrypt and open this message again
5) text colors are incorrect

<img src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/1cac3fb1-b681-48c2-959c-4d91474e0989" width="400">

Reply:
Do you know good way to reload view when traitCollectionDidChange?
I don't know and I'm thinking to override traitCollectionDidChange for view controller and call `updateForNewTraitCollection` and change background/font color for every nodes.

Reply:
I tried to replace `.foregroundColor: Constants.textColor` with `.foregroundColor: UIColor.mainTextColor` and it fixed this issue. Maybe `Constants` is initialized only on first opening and that's why color doesn't change.

Reply:
Sounds interesting. But as you know, content view, pass phrase input doesn't use Constants and those colors don't change when color schema changes.

Reply:
Yeah, it's strange, as other colors in the app change without issues

Reply:
I think It's because other views listen to `traitCollectionDidChange` and call `reloadData` in `tableNode`

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/blob/2c3a563374ece89c69bb70dfdca8c88d49e550e9/FlowCryptUI/Nodes/TableViewController.swift#L22

Reply:
But AFAIK, there's no direct way to reload ASDisplayNode

Reply:
> But as you know, content view, pass phrase input doesn't use Constants and those colors don't change when color schema changes.

I tested it - passphrase input color changes correctly, so I think changing `.foregroundColor: Constants.textColor` to `.foregroundColor: UIColor.mainTextColor` should be enough for fixing color issues.

And let's also change separator color from `UIColor.lightGray` to `UIColor.separator`, as currently it looks too bright in dark mode.

Reply:
OK